Package.swift

```diff
@@ -5,14 +5,14 @@ let package = Package(
     name: "MapLibre Native",
     products: [
         .library(
-            name: "Mapbox",
-            targets: ["Mapbox"])
+            name: "MapLibre",
+            targets: ["MapLibre"])
     ],
     dependencies: [
     ],    
     targets: [
         .binaryTarget(
-            name: "Mapbox",
+            name: "MapLibre",
```

Comment:
The line below still has mapbox in it...

Reply:
Yes but that will be auto-replaced when a release is made.